diff --git a/node_modules/standard-version/node_modules/conventional-changelog-conventionalcommits/templates/header.hbs b/node_modules/standard-version/node_modules/conventional-changelog-conventionalcommits/templates/header.hbs
index a4d6ec4..fdb6d7b 100644
--- a/node_modules/standard-version/node_modules/conventional-changelog-conventionalcommits/templates/header.hbs
+++ b/node_modules/standard-version/node_modules/conventional-changelog-conventionalcommits/templates/header.hbs
@@ -1,11 +1,23 @@
 {{#if isPatch~}}
-  ###
+##
 {{~else~}}
-  ##
+#
 {{~/if}} {{#if @root.linkCompare~}}
-  [{{version}}]({{compareUrlFormat}})
+[{{version}}](
+{{~#if @root.repository~}}
+{{~#if @root.host}}
+{{~@root.host}}/
+{{~/if}}
+{{~#if @root.owner}}
+{{~@root.owner}}/
+{{~/if}}
+{{~@root.repository}}
+{{~else}}
+{{~@root.repoUrl}}
+{{~/if~}}
+/compare/{{previousTag}}...{{currentTag}})
 {{~else}}
-  {{~version}}
+{{~version}}
 {{~/if}}
 {{~#if title}} "{{title}}"
 {{~/if}}
